src/material-experimental/mdc-chips/testing/chip-harness.spec.ts

```diff
@@ -34,22 +35,37 @@ describe('MatChipHarness', () => {
     expect(await harnesses[4].getText()).toBe('Chip Row');
   });
 
-  it('should be able to remove a mat-chip-row', async () => {
+  it('should be able to remove a chip', async () => {
     const removeChipSpy = spyOn(fixture.componentInstance, 'removeChip');
 
     const harnesses = await loader.getAllHarnesses(MatChipHarness);
     await harnesses[4].remove();
 
     expect(removeChipSpy).toHaveBeenCalledTimes(1);
   });
+
+  it('should get the disabled state of a chip', async () => {
+    const harnesses = await loader.getAllHarnesses(MatChipHarness);
+    const disabledStates = await Promise.all(harnesses.map(harness => harness.isDisabled()));
```

Comment:
Please prefer the `parallel` method from cdk/testing over `Promise.all` (I've been meaning to do a pass on this). It will optimize change detection so its only run once rather than once-per-item